caom2pipe/execute_composable.py

```diff
@@ -1180,18 +1170,11 @@ def do_one(self, storage_name):
         return result
 
 
-def decompressor_factory(
-    config, working_directory, log_level_as, storage_name
-):
-    if config.features.supports_decompression:
-        if config.collection == 'CFHT':
-            return FitsForCADCCompressor(
-                working_directory, log_level_as, storage_name
-            )
-        else:
-            return FitsForCADCDecompressor(working_directory, log_level_as)
+def decompressor_factory(config, working_directory, log_level_as, storage_name):
```

Comment:
I think it's required in perpetuity, since providing cutouts is a thing CADC does, not the observatories.

If space is an issue, there's always `fpack`. Do you want to check with Séverin?